### DISCUSSION TITLE
Open OME webRTC stream with OBS

### DISCUSSION THREAD

**Discussion by farbhaus** [OvenMediaLabs/OvenMediaEngine#1521] [Help]
Hello I found this video showcasing a p2p srt stream with OBS:
https://www.youtube.com/watch?v=eDgZ-IqvCJc&ab_channel=2NUProductions

I use OME with Davinci Resolve to send my video signal to clients. Some clients have the ability to view my stream on a broadcast montior with the use of a Blackmagic decklink card so I was wondering if it's possible to open a OME stream directly in OBS, which can make use of decklink/ultrastudio SDI cards.

**Comment by bchah** [comment]
You can push SRT to OBS from OME, if you set up an OBS media source as SRT listener. 

I know this is low priority for the roadmap, but this is just one more example where an SRT Listener Publisher would really help us! Otherwise for SRT transport *out of* OME you'll need to operate your target device in Listener mode and use the Push API for now.

EDIT: for example, add a Media Source, uncheck 'Local File' and construct an SRT listener URL like "srt://YOURIP:9999?mode=listener". In my experience pushing SRT to the same host does not work so try pushing from another OME instance on your LAN.

**Comment by farbhaus** [comment]
I\ll give that a shot thanks!
